deployment/bentoml/mednist_classifier_bentoml.ipynb

```diff
@@ -94,6 +94,12 @@
     }
    ],
    "source": [
+    "from bentoml.utils import cached_property  # noqa: F401\n",
```

Comment:
This code was removed from a "writefile" cell and moved here. It shouldn't be here but is needed in the file the cell is meant to write, so whatever formatting was done has also here ignored "writefile" and treated that cell like a normal Python cell.